lib/fx/adapters/postgres/functions.rb

```diff
@@ -53,6 +61,25 @@ def functions_from_postgres
         def to_fx_function(result)
           Fx::Function.new(result)
         end
+
+        def schema_aware_name(function)
```

Comment:
I considered moving both of these "schema-aware" mappers to `Fx::Function`, but it also complicates the tests for that class in ways that didn't feel necessary. (I figured I would leave it up to you @teoljungberg, as to whether that class should be a "dumb" wrapper or a "smart" translator.)